FAQ/faq10.md

```diff
@@ -0,0 +1,3 @@
+# How does this work relate to the work in other organizations?
```

Comment:
For now suggest to use Title Case for such titles (maybe we later change to expose them more like the questions they are).